### ISSUE TITLE
tvtk.Points ``__getitem__`` does not accept python long integers

### ISSUE THREAD

**Issue by itziakos** [enthought/mayavi#173]
The following example demonstrates the issue

```
In [10] points = tvtk.Points()

In [11]: points.insert_next_point((0,1,2))
Out[11]: 0L

In [12]: points[0]
Out[12]: (0.0, 1.0, 2.0)

In [13]: points[0L]  # raises TypeError
```

**Comment by itziakos** [comment]
*Update*: The wrapped vtk object does accept `long`

```python
In [40]: a = points._vtk_obj

In [41]: a.GetPoint(0L)
Out[41]: (0.0, 1.0, 2.0)

```

**Comment by dmsurti** [comment]
Fixed in #223. Then only change is for index look up. And, `get_point` works as expected with `long` in tvtk. This has been added to test case in PR

```
In [10]: pts.get_point(0L)
Out[10]: (0.0, 1.0, 2.0)
```
